meld/system/builders/build_elastic_network_restraints.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright 2015 by Justin MacCallum, Alberto Perez, Ken Dill
+# All rights reserved
+#
+
+"""
+Methods to read template coordinates from a system object and build elastic network restraints prior to simulation for a specific chain.
+"""
+
+import numpy as np  # type: ignore
+from scipy.spatial.distance import pdist  # type: ignore
+from scipy.spatial.distance import squareform  # type: ignore
+from typing import List, Optional, Union
+
+from meld.system import restraints
+from meld.system import meld_system
+from meld.system import scalers
+from meld import unit as u
+
+def create_elastic_network_restraints(
+    system: meld_system.System,
+    scaler: scalers.RestraintScaler,
+    chain_index: int,
```

Comment:
I was thinking that chain_index could also be an iterable. So you could either specify a single chain, or multiple chains. In the case of multiple chains, you would be creating the joint set of restraints between both chains, e.g. to keep a multimer together. If you wanted to separately create restraints for each chain, you would just call this function multiple times, once with each chain.

Reply:
I've come up with something that works for specifying multiple chain indices. I'm not sure if it's the best/most pythonic way of doing things, so please let me know if there is a better way.